Form-Controls/index.html

```diff
@@ -13,15 +13,46 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label>Name</label>
```

Comment:
`<label>` is missing an attribute. Can you find what you're missing? 👀 https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label#attributes